go.mod

```diff
@@ -20,6 +19,7 @@ require (
 	github.com/boltdb/bolt v1.3.2-0.20180302180052-fd01fc79c553 // indirect
 	github.com/bwmarrin/discordgo v0.19.1-0.20190513173526-8d8906ce4b95
 	github.com/carlosdp/twiliogo v0.0.0-20161027183705-b26045ebb9d1
+	github.com/cespare/xxhash/v2 v2.1.1
```

Comment:
I expected old library to disappear from dependencies, but it did not. Maybe run `go mod tidy` or something.